### ISSUE TITLE
Request: add forecast_reference_time as coordinate

### ISSUE THREAD

**Issue by aaronspring** [ecmwf-lab/climetlab-s2s-ai-competition#3]
Thanks for make ECMWF output so easily available with this package.

I am lacking explicit information about the `forecast_reference_time`, which I think is the time, when the model was started/initialized, which is mentioned in the attrs of `steps` `forecast_reference_time = forecast_time - step`. In multi-dimensional zarr, forecast_reference_time would be 2-dimensional: `forecast_reference_time(forecast_time, step)`.

Would it be possible to add a new coordinate to the output, specifically `zarr`:

![image](https://user-images.githubusercontent.com/12237157/111763835-dbe17f00-88a2-11eb-8263-6e7abe899578.png)

**Comment by aaronspring** [comment]
I just learned that `forecast_time` is the time of initialization of the model (simulation start time). Could this information go into the `forecast_time.attrs`. I would also rename to something more specific like `initialization_time`, `init_time` or `init`. Isnt there a CF standard for this?

didnt see `time` in `data_vars`: `.set_coords('time')` does the trick.

**Comment by aaronspring** [comment]
still for the uploaded data I think having `valid_time` as a coordinate and not data variable would be better @floriankrb

**Comment by aaronspring** [comment]
Now I see you changed climetlab to have it as a coordinate and not data_var. I would have rather updated as coordinate in the file in the cloud.

**Comment by floriankrb** [comment]
I agree, thank you for checking this and reopening the issue. I fixed it with new data and updated the README.

**Comment by floriankrb** [comment]
This does not seem to be actually fixed. I will need to work more on it.

**Comment by floriankrb** [comment]
I uploaded version 0.2.5 of the data with this issue fixed.
